### DISCUSSION TITLE
docker images not complete

### DISCUSSION THREAD

**Discussion by dislazy** [dani-garcia/vaultwarden#1368] [Q&A]
[root@XXX ~]# docker pull bitwardenrs/server:testing
Trying to pull repository docker.io/bitwardenrs/server ... 
testing: Pulling from docker.io/bitwardenrs/server
6ec7b7d162b2: Already exists 
4065a816fc92: Pulling fs layer 
9561e83017d6: Pulling fs layer 
470b918337e3: Pulling fs layer 
ea083e1e387b: Waiting 
39f3a1c871e5: Waiting 
882868ed9dea: Waiting 
63510ec32190: Waiting 
error pulling image configuration: received unexpected HTTP status: 500 Internal Server Error

**Comment by jjlin** [comment]
Works for me:
```
$ docker pull bitwardenrs/server:testing
testing: Pulling from bitwardenrs/server
a076a628af6f: Pull complete
1d25e452c1f2: Pull complete
1a3e8e8cf8e8: Pull complete
2331c5fc344d: Pull complete
5e6c68a1d4e0: Pull complete
b451faa48835: Pull complete
c6cf33363b56: Pull complete
4f3dfa1a981c: Pull complete
Digest: sha256:26de5f7bd4fa120f9c92d1d8c8cac45b104f76f4d7cc79b8089defa78d084fc5
Status: Downloaded newer image for bitwardenrs/server:testing
docker.io/bitwardenrs/server:testing
```
You're not getting the current layers either, so maybe you have some kind of caching or something else in between you and Docker Hub.

**Comment by BlackDex** [comment]
Or you are hitting the docker pull limit somewhere?
